velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -70,6 +74,15 @@ TEST_F(GreatestLeastTest, nanInput) {
       "Invalid argument to greatest(): NaN");
 }
 
+TEST_F(GreatestLeastTest, tryWithNan) {
+  runTest<double>(
+      "try(least(c0, 1.0))", {{0, 1.1, std::nan("1")}}, {0, 1.0, std::nullopt});
+  runTest<double>(
+      "try(greatest(c0, 1.0))",
+      {{0, 1.1, std::nan("1")}},
```

Comment:
Are the input vectors for the 2 cases the same? Consider extract an 'input' temporary variable.